### PR TITLE
Release 10.6.3

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-6-3.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+id: ""
+sidebar_label: "Release v10.6.3"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Improvements
+
+---
+
+- Introduced a new option, `Show At`, in the Prefab UI configuration. When using a Prefab in another application or a Prefab, all Prefab elements' UI properties display by default when you select the Prefab element. Note, when you choose a widget from the `Show At` list, the UI property will display when you select that UI property.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-6-3.md:12]: @nikhilesh-kv are the other prefab improvements you did in this release? Can we mention them here?

File: learn/wavemaker-release-notes/v10-6-3.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+- Fixed an issue related to querying a large string data from DB shortened in the variable's `successCallback()`.
+- Fixed an issue related to the placeholder of the Text widget.
+- Fixed an issue with the dynamic display format for the Text widget.
+- Fixed an issue related to console errors in Prefab when using it multiple times in a Live Form.
+- Fixed an issue with the dropdown disabled property of the Autocomplete widget in DataTable.
+- Fixed an issue related to the AnimationInterval property of the Carousel widget when set through the script.
+- Fixed an issue with missing digits in the Number widget when editing the value.
+- Fixed an issue related to Fields overlapping when updating to Material-2.0.  
+**Note**: Adding fixed heights lead to CSS bloating. Therefore, it loses its responsiveness. We advise not to restrict the grid rows with any strict boundaries.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-6-3.md:25]: CSS bloating needs rewording @lovinahmed

File: learn/wavemaker-release-notes/v10-6-3.md (hunk 1 above)
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-6-3.md:12]: @swethakundaram  Form layout improvements done by @aparnasj also need to be mentioned here

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

File: learn/wavemaker-release-notes/v10-6-3.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ sidebar_label: "Release v10.6.3"
 
 - Introduced a new option, `Show At`, in the Prefab UI configuration. When using a Prefab in another application or a Prefab, all Prefab elements' UI properties display by default when you select the Prefab element. Note, when you choose a widget from the `Show At` list, the UI property will display when you select that UI property.
 - Included an option for Number and Currency widgets to limit the decimal value entry.
+- Improved the Form interaction to include a Floating button for Inline layout of Form Widget.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-6-3.md:14]: doesn't sound right. @aparnasj your inputs here

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-6-3.md (hunk 3 above)
Comment (vibhus) [reply learn/wavemaker-release-notes/v10-6-3.md:14]: Introduced floating labels for form widgets to give Material Design like experience for form fields.

File: learn/wavemaker-release-notes/v10-6-3.md (hunk 2 above)
Comment (lovin26) [reply learn/wavemaker-release-notes/v10-6-3.md:26]: Adding Fixed heights lead to a rigid CSS structure.

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

File: learn/wavemaker-release-notes/v10-6-3.md (hunk 3 above)
Comment (prasanth-reddy-wavemaker-com) [reply learn/wavemaker-release-notes/v10-6-3.md]: added

File: learn/wavemaker-release-notes/v10-6-3.md (hunk 1 above)
Comment (prasanth-reddy-wavemaker-com) [reply learn/wavemaker-release-notes/v10-6-3.md]: I added it in the improvements section. Take a look.

**Pull request merged**